attributes/default.rb

```diff
@@ -4,3 +4,6 @@
 default['osl-mysql']['xtrabackuprb']['version'] = '0.0.9'
 default['osl-mysql']['backup_dir'] = '/data/backup'
 default['osl-mysql']['enable_percona_client'] = false
+
+# Lock mysql version until upstream is fixed
+default['percona']['version'] = '5.7' if node['platform_version'].to_i > 7
```

Comment:
Please look at the default recipe..this isn't the correct way to do it. Also you'll need to update the tests.